src/studio/src/designer/frontend/app-development/sharedResources/user/userSlice.ts

```diff
@@ -31,6 +39,22 @@ export interface IUserActionRejected {
   error: Error;
 }
 
+export interface IFetchRemainingSessionFulfilled {
+  remainingMinutes: number;
+}
+
+export interface IFetchRemainingSessionRejected {
```

Comment:
You could just use `IUserActionRejected`, since it's the same content?
The idea was that any action errors result in the same payload type, so they might as well use the same interface as well.

Reply:
Agreed. Fixed.